src/app/services/session.service.ts

```diff
@@ -67,6 +67,11 @@ export class SessionService {
     }
   }
 
+  setSession(customer, vendor){
```

Comment:
This should be named better. The code within this function name is not setting a session. It's updating the names of the customer and vendor. Update the function name to better represent what's going on inside.